src/transxchange/read.rs

```diff
@@ -442,7 +562,11 @@ fn load_routes_vehicle_journeys_calendars(
         let line_ref = vehicle_journey.try_only_child("LineRef")?.text();
         let vehicle_journey_code = vehicle_journey.try_only_child("VehicleJourneyCode")?.text();
         let id = format!("{}:{}:{}", service_ref, line_ref, vehicle_journey_code);
-        let calendar = create_calendar_dates(transxchange, vehicle_journey)?;
+        let dates = create_calendar_dates(transxchange, vehicle_journey)?;
+        let calendar = Calendar {
+            id: format!("CD:{}", id),
```

Comment:
We keep CD or not ? :confused:

Reply:
ok to keep it like this :)

Reply:
@woshilapin In my opinion we should skip the vj if there is no date (like for stop_times).

Reply:
Done in 6741ad303819e91e656e07f41c79a44ae4b9212f.